src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,576 @@
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0024" side="server" define="MIC_STATUS" type="boolean" writable="true" optional="true">
+      <description>MicStatus</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0025" side="server" define="MIC_VOLUME_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>MicVolumeLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0026" side="server" define="MIC_MAX_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>MicMaxLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0027" side="server" define="MIC_MIN_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>MicMinLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0028" side="server" define="MIC_AGCENABLED" type="boolean" writable="true" optional="true">
+      <description>MicAGCEnabled</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0029" side="server" define="IMAGE_ROTATION" type="int16u" max="359" optional="true">
+      <description>ImageRotation</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x002A" side="server" define="IMAGE_FLIP_HORIZONTAL" type="boolean" optional="true">
+      <description>ImageFlipHorizontal</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x002B" side="server" define="IMAGE_FLIP_VERTICAL" type="boolean" optional="true">
+      <description>ImageFlipVertical</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x002C" side="server" define="LOCAL_VIDEO_RECORDING_ENABLED" type="boolean" writable="true" optional="true">
+      <description>LocalVideoRecordingEnabled</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x002D" side="server" define="LOCAL_SNAPSHOT_RECORDING_ENABLED" type="boolean" writable="true" optional="true">
+      <description>LocalSnapshotRecordingEnabled</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x002E" side="server" define="STATUS_LIGHT" type="boolean" writable="true" optional="true">
+      <description>StatusLight</description>
```

Comment:
OK, if this stays, the same spec comment: What do "true" and "false" mean for an attribute called StatusLight?  Is this about whether it's on?  About whether it's _allowed_ to be on?  Something else entirely?

Needs better naming.

Reply:
Captured comment in spec.